exporter/cassandraexporter/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/cassan
 
 go 1.21
 
-toolchain go1.21.6
+toolchain go1.21.7
```

Comment:
How come this specific exporter needs the toolchain?

Reply:
No idea! :confused:

Reply:
It was added via https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31109.  @codeboten do you know? If it is removed does stuff break?

Reply:
it's unclear to me why this exporter needs it. I'd suspect one of its dependencies causes this?

Reply:
Let me delete the line and see what happens :smile: